spec/views/content_assets/show.html.erb_spec.rb

```diff
@@ -2,9 +2,7 @@
 
 RSpec.describe "content_assets/show", type: :view do
   before(:each) do
-    @content_asset = assign(:content_asset, ContentAsset.create!(
-                                              title: "Title",
-                                            ))
+    @content_asset = FactoryBot.create(:content_asset)
```

Comment:
Can we move these variables into let definitions
https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let